src/components/modals/login/utils.tsx

```diff
@@ -0,0 +1,108 @@
+import { useLinkEvmAddress } from '@/services/subsocial/evmAddresses/mutation'
+import { getSubsocialApi } from '@/subsocial-query/subsocial/connection'
+import { decodeAddress } from '@polkadot/keyring'
+import { BN, u8aToHex } from '@polkadot/util'
+import { useEffect, useState } from 'react'
+import { useDisconnect, useSignMessage } from 'wagmi'
+
+export const buildMsgParams = async (substrateAddress: string) => {
```

Comment:
don't export it, because its only used internally right?
same with `useSignEvmLinkMessage`